store/iavlstore.go

```diff
@@ -16,18 +16,19 @@ import (
 const (
 	defaultIAVLCacheSize  = 10000
 	defaultIAVLNumRecent  = 100
-	defaultIAVLStoreEvery = 1
+	defaultIAVLStoreEvery = 10000
```

Comment:
Can we choose one strategy (`syncable`) as default instead of duplicating constants?